bfd/src/main/java/gov/cms/ab2d/bfd/client/BFDClientImpl.java

```diff
@@ -55,7 +58,9 @@ public BFDClientImpl(IGenericClient bfdFhirRestClient) {
      */
     @Override
     public Bundle requestEOBFromServer(String patientID) {
-        log.debug("Attempting to fetch EOBs for patient ID {} from baseURL: {}", patientID,
+        MDC.put(PATIENT_ID_LOG, patientID);
```

Comment:
Let's drop this one (and any other logging patient ID) to avoid accidentally logging PII.